roles/airflow/templates/airflow-webserver.service.j2

```diff
@@ -7,7 +7,7 @@ EnvironmentFile=/home/airflow/airflow-environment
 User=airflow
 Group=airflow
 Type=simple
-ExecStart=/bin/bash -lc 'source /home/airflow/anaconda3/bin/activate o3; airflow webserver -p 8080'
+ExecStart=/bin/bash -l -c 'conda activate o3 && airflow webserver -p 8080'
```

Comment:
This does not work on Ubuntu did you test if it works on CentOS? 

```
● airflow-webserver.service - Airflow webserver daemon
   Loaded: loaded (/etc/systemd/system/airflow-webserver.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Tue 2019-01-29 20:59:13 UTC; 3s ago
  Process: 28228 ExecStart=/bin/bash -l -c conda activate o3 && airflow webserver -p 8080 (code=exited, status=127)
 Main PID: 28228 (code=exited, status=127)

Jan 29 20:59:13 ip-172-30-3-126 systemd[1]: Started Airflow webserver daemon.
Jan 29 20:59:13 ip-172-30-3-126 bash[28228]: /home/airflow/.profile: line 35: conda: command not found
Jan 29 20:59:13 ip-172-30-3-126 bash[28228]: /bin/bash: conda: command not found
```

I can assure you that `source /home/airflow/anaconda3/bin/activate o3` works on all environments.

Reply:
This is solved by the `bash -ilc` fix